### DISCUSSION TITLE
Put `jobs` at the end the file

### DISCUSSION THREAD

**Discussion by benkeil** [typesafegithub/github-workflows-kt#621] [Polls]
At the moment all properties are sorted alphabetically.
In my opinion it is easier to read if you see the "settings" first and the jobs last.
So I propose to put `jobs` at the end. It could be configurable.

**Comment by krzema12** [comment]
Hi @benkeil! Can you give an example? In general, only custom arguments are put after jobs:

https://github.com/krzema12/github-workflows-kt/blob/444d0cf5137c116f7ded8eefa0f734ef6cb37eac/library/src/main/kotlin/it/krzeminski/githubactions/yaml/ToYaml.kt#L103-L114

which actually could be changed, but it doesn't match what you write about alphabetic order. Let's clarify it before I make any changes.

**Reply by krzema12** [reply]
https://github.com/krzema12/github-workflows-kt/pull/624 - I moved custom arguments before jobs and steps.

**Reply by benkeil** [reply]
sorry for late response, I was in vacation. If the order in the code also affects the order of the resulting yaml than it's great.
